src/plugins/pcr/src/FindPrimerPairsWorker.h

```diff
@@ -79,7 +84,8 @@ class FindPrimerPairsWorkerFactory : public DomainFactory {
 class FindPrimersTask : public Task {
     Q_OBJECT
 public:
-    FindPrimersTask(const QString& outputFileUrl, const QList<DNASequence>& sequences);
+    FindPrimersTask(const QString& outputFileUrl, const QList<DNASequence>& sequences, BaseTempCalc* temperatureCalculator);
+    ~FindPrimersTask();
```

Comment:
I see a lot of boilerplate code do deallocate 'temperatureCalculator' across the code. 
Use shared pointer?

Reply:
OK